README.md

```diff
@@ -1,2 +1,51 @@
 # playbook
-collection of venturemark playbooks
+
+Collection of playbooks for the Venturemark platform.
+
+
+
+### What Is A Playbook?
+
+A playbook aims to provide you with inspiration and instruction for how to use
+the Venturemark platform. The Venturemark platform provides you with an abstract
+and powerful way to combine contextual information and factual data. Due to the
+abstract nature of the Venturemark platform, you can use it in many different
+ways. There are many powerful use cases for high performing individuals to use
+the Venturemark platform. A playbook is merely a piece of documentation
+providing you with instructions for habit forming processes. These processes aim
+to achieve a track record of better results throughout your venture's journey.
+
+
+
+### How To Use A Playbook?
+
```

Comment:
Concise: 

With consistency. Input daily for 1 week to form a habit. This habit forms a track record. This track record builds insights. These insights make you better.

Reply:
I did not put any cadence here because this heavily depends on the playbook. There I wanted to provide suggestions about how to approach the interval. If you follow a playbook of monthly recurring users, you do not need to write create a metric update every day, since the numbers do almost not change at all at this scale. There are playbooks though where you should create metric updates every day. So, consistency is relative to your currently relevant context.

Reply:
I agree with the logic. As I was reading, I wondered what the purpose of this text was (FAQ, About, Support Documentation) as it would help me better understand correct format. From marketing point - it's important to direct language towards a specific segment or use case.